engine/src/main/java/org/hibernate/validator/HibernateValidatorConfiguration.java

```diff
@@ -62,6 +66,16 @@
 	String CONSTRAINT_MAPPING_CONTRIBUTORS = "hibernate.validator.constraint_mapping_contributors";
 
 	/**
+	 * Property for configuring script evaluator factory, allowing to set up which factory will be used to create
+	 * {@link ScriptEvaluator} for evaluation of script expressions in
+	 * {@link ScriptAssert} and {@link ParameterScriptAssert}
+	 * constraints. A fully qualified name of a class implementing  {@link ScriptEvaluatorFactory} is expected as a value.
+	 *
+	 * @since 6.1
```

Comment:
On the question of the version to introduce this, my proposal is this:

* Let's add it in 6.0.3, marking all the members as `@Experimental` or `@Incubating`
* Promote to non-experimental in the future, e.g. 6.1 or a bit later, based on experiences
* Alter `@since` to 6.0.3 :)

I.e. let's bend the semantic versioning scheme a bit and take the liberty to introduce incubating features in a micro.

Reply:
Done.